modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -29,6 +34,12 @@ The Red Hat Telco RAN {product-version} solution has been validated using the fo
 |SRIOV Operator
 |4.14
 
+|Node Tuning Operator
+|4.14
+
```

Comment:
NTO is part of the default OCP installation. Do we need to include it here? (I'm ok if we do)

Reply:
I don't think it's doing any harm and it is a critical part of the DU profile so let's leave it in. Thanks.